### PR TITLE
Fix clang tidy issues in CCB

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: vowpalwabbit/conditional_contextual_bandit.cc [hunk 1]
```diff
@@ -270,7 +271,7 @@ void remove_slot_features(example* shared, example* slot)
       continue;
     }
 
-    else if (index == default_namespace)  // slot default namespace has a special namespace in shared
```
Comment (peterychang) [inline vowpalwabbit/conditional_contextual_bandit.cc]: I guess it doesn't matter either way, but why remove this?

File: vowpalwabbit/conditional_contextual_bandit.cc [hunk 2]
```diff
@@ -424,9 +434,13 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
     build_cb_example<is_learn>(data.cb_ex, slot, data);
 
     if (data.all->audit)
-      inject_slot_id<true>(data, data.shared, slot_id);
+    {
+      inject_slot_id<true>(data, data.shared, slot_id);\
```
Comment (peterychang) [inline vowpalwabbit/conditional_contextual_bandit.cc]: remove the \

File: vowpalwabbit/conditional_contextual_bandit.cc (hunk 1 above)
Comment (jackgerrits) [reply vowpalwabbit/conditional_contextual_bandit.cc]: The specific clang-tidy check here is to not follow a `continue` statement with an `else if` as it is kind of misleading

File: vowpalwabbit/conditional_contextual_bandit.cc (hunk 2 above)
Comment (jackgerrits) [reply vowpalwabbit/conditional_contextual_bandit.cc]: Oops! Thanks

Comment (jackgerrits) [reply vowpalwabbit/conditional_contextual_bandit.cc]: Oops! Thanks

**Pull request merged**